### PR TITLE
Wip jurisdiction endpoint

### PR DESCRIPTION
**Pull request by jafow**
# summary

adds endpoints for `WipJurisdiction` data, and some scripts etc for devving. 

## todo
- [ ] locations as their own resource

i think that can be pulled out no problem still

### CODE REVIEW

File: backend/controllers/jurisdictions.js [hunk 1]
```diff
@@ -16,3 +29,104 @@ exports.delete = (req, res, next) => {
...
+    return handleError(err, 400, res)
+  }
+}
+
+/**
+ * update a wip jurisdiction
+ */
+exports.updateWip = async (req, res) => {
+  let wipJurisdictionId = req.params.wipJurisdictionId
+  if (wipJurisdictionId === undefined || wipJurisdictionId === null) {
```
Comment (jafow) [inline backend/controllers/jurisdictions.js:105]: some of this kinda stuff we can get rid of with a validation middleware; it's on my todo list

File: backend/_misc/requests/Makefile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+PHONY: listWip createWip auth listWipLocation
```
Comment (jafow) [inline backend/_misc/requests/Makefile:1]: dunno if this is helpful to anyone but me, and i'll take it out soon as we've stabilized a bit. adding in lieu of making actual tests or script for calling the API.

Comment (jmensch1) [reply backend/_misc/requests/Makefile:1]: sounds good! In the 311 project we're using Postman/Newman for api testing, which integrates nicely with CI and is pretty user-friendly. But given the timeframe here this approach makes sense.

File: backend/routes/jurisdictions.js [hunk 3]
```diff
@@ -6,4 +6,17 @@ router.get('/', control.list)
...
+router.post('/wip', auth(['volunteer', 'admin']), control.createWip)
+router.put(
+  '/wip/:wipJurisdictionId',
+  auth(['volunteer', 'admin']),
+  control.updateWip
+)
+
+router.get(
+  '/wip/:wipJurisdictionId/locations',
+  control.listWipJurisdictionsWipLocation
```
Comment (jmensch1) [inline backend/routes/jurisdictions.js:19]: noticed no auth restrictions on the wip GET requests? I'm fine with that, just wanted to make sure it was intentional

Comment (jmensch1) [reply backend/routes/jurisdictions.js:19]: I'm gonna merge so I can play with this! can add auth later if necessary

**Review by jmensch1 — APPROVED** [review]

**Pull request merged**